logging/simplelog/simplelog.go

```diff
@@ -0,0 +1,134 @@
+// iample simplelog writes some entries, lists them, then deletes the log.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	// [START imports]
+	// NOTE: This will become cloud.google.com/go/logging soon.
+	"cloud.google.com/go/preview/logging"
+
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+	// [END imports]
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Println("Usage: simplelog <project-id>")
```

Comment:
We use an env variable for the project name in some references, an argument in some other. Why not using GCLOUD_PROJECT env var everywhere?